### PR TITLE
Update kubevirt on minikube quickstart page

### PR DESCRIPTION
**Pull request by mazzystr**
**What this PR does / why we need it**:
Update kubevirt on minikube quickstart page

### CODE REVIEW

**Comment by hroyrh** [comment]
@mazzystr seems like the ci-build is failing due to words not included in whitelist

File: pages/quickstart_minikube.md [hunk 1]
```diff
@@ -4,186 +4,124 @@ title: KubeVirt Quickstart with Minikube
...
-Now, set the VM driver to KVM2:
+    > info "Minikube handbook"
+    > See the minikube handbook [_here_](https://minikube.sigs.k8s.io/docs/) for advanced start options and instructions on how to operate minikube.
 
+* A kubectl client is necessary for operating a Kubernetes cluster. Minikube ships a  kubectl client version that matches the kubernetes version to avoid issues regarding [_skew_](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew).<br><br>
+To use the minikube shipped client all normal `kubectl` commands should be performed as `minikube kubectl`.<br><br>
+Or the shipped client can be added to aliases by running the following:
 ```bash
-{% include scriptlets/quickstart_minikube/02_minikube_config_vm_driver.sh -%}
+alias kubectl='minikube kubectl'
```
Comment (davidvossel) [inline pages/quickstart_minikube.md:39]: this alias doesn't work for me.

the syntax is `minikube kubectl -- ` then all the kubectl args.

for example, this fails

```
[dvossel@localhost minikube]$ alias kubectl='minikube kubectl' 
[dvossel@localhost minikube]$ kubectl get pods -n kubevirt
Error: unknown shorthand flag: 'n' in -n
See 'minikube kubectl --help' for usage.
```

but this works. 

```
[dvossel@localhost minikube]$ alias kubectl='minikube kubectl -- ' 
[dvossel@localhost minikube]$ kubectl get pods -n kubevirt
NAME                              READY   STATUS    RESTARTS   AGE
virt-api-5f8b7c7fcc-2v6vq         1/1     Running   0          9m47s
virt-api-5f8b7c7fcc-fjw62         1/1     Running   0          9m47s
virt-controller-565b5d87d-28wph   1/1     Running   0          9m27s
virt-controller-565b5d87d-wh4hg   1/1     Running   0          9m27s
virt-handler-jzbff                1/1     Running   0          9m27s
virt-operator-6454599c8c-tvrcq    1/1     Running   0          10m
virt-operator-6454599c8c-wwjcf    1/1     Running   0          10m
```

Comment (mazzystr) [reply pages/quickstart_minikube.md:39]: oh my gosh!   thanks for that catch!  ...and thanks for actually running through the lab.

**Comment by davidvossel** [comment]
/lgtm
/approve

i don't know if i'm an approver, but i'll try :D

**Pull request merged**